react/src/Analyses/index.tsx

```diff
@@ -301,6 +332,37 @@ export default function Analyses() {
                             editable: "never",
                             width: "8%",
                         },
+                        {
+                            title: "Priority",
+                            field: "priority",
+                            type: "string",
+                            width: "8%",
+                            lookup: priorityLookup,
+                            editComponent: ({ onChange, value }) => {
+                                //react doesn't like default null values in forms so we have to workaround
+                                const cast = value ? value : "None";
+                                return (
+                                    <Select
+                                        value={cast}
```

Comment:
You should be able to pass `value || "None"` here instead then

Reply:
updated in [4618fc9](https://github.com/ccmbioinfo/stager/pull/522/commits/4618fc9cbced57960989026ffdefaffd90100b79#diff-19e03fc8aab473e13bc8ddf811cf174a6b4c23b5c0740147da368a1478089cc6R359)